src/Bridges/Dibi/Dibi3Adapter.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * This file is part of the Nextras community extensions of Nette Framework
+ *
+ * @license    New BSD License
+ * @link       https://github.com/nextras/migrations
+ */
+
+
+namespace Nextras\Migrations\Bridges\Dibi;
+
+use DateTime;
+use Dibi\Connection as DibiConnection;
+use Nextras\Migrations\IDbal;
+
+class Dibi3Adapter extends DibiAdapter implements IDbal
```

Comment:
ok and one last thing, do not extend the original driver, make it indenpendent :-)